### ISSUE TITLE
Issue when running BedrockConnect jar on machine with Minecraft server

### ISSUE THREAD

**Issue by rstone13** [Pugmatt/BedrockConnect#160]
I'm attempting to set up BedrockConnect on my Ubuntu 18.04.5 machine and I got the following error:

`
java.util.concurrent.CompletionException: io.netty.channel.unix.Errors$NativeIoException: bind(..) failed: Address already in use
        at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:292)
        at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:308)
        at java.util.concurrent.CompletableFuture.biRelay(CompletableFuture.java:1298)
        at java.util.concurrent.CompletableFuture$BiRelay.tryFire(CompletableFuture.java:1284)
        at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:488)
        at java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:1990)
        at com.nukkitx.network.util.Bootstraps.lambda$allOf$0(Bootstraps.java:111)
        at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:577)
        at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:551)
        at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:490)
        at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:615)
        at io.netty.util.concurrent.DefaultPromise.setFailure0(DefaultPromise.java:608)
        at io.netty.util.concurrent.DefaultPromise.tryFailure(DefaultPromise.java:117)
        at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetFailure(AbstractChannel.java:994)
        at io.netty.channel.AbstractChannel$AbstractUnsafe.bind(AbstractChannel.java:553)
        at io.netty.channel.DefaultChannelPipeline$HeadContext.bind(DefaultChannelPipeline.java:1346)
        at io.netty.channel.AbstractChannelHandlerContext.invokeBind(AbstractChannelHandlerContext.java:503)
        at io.netty.channel.AbstractChannelHandlerContext.bind(AbstractChannelHandlerContext.java:488)
        at io.netty.channel.DefaultChannelPipeline.bind(DefaultChannelPipeline.java:985)
        at io.netty.channel.AbstractChannel.bind(AbstractChannel.java:247)
        at io.netty.bootstrap.AbstractBootstrap$2.run(AbstractBootstrap.java:344)
        at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163)
        at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:510)
        at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:413)
        at io.netty.util.concurrent.SingleThreadEventExecutor$6.run(SingleThreadEventExecutor.java:1050)
        at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
        at java.lang.Thread.run(Thread.java:748)
Caused by: io.netty.channel.unix.Errors$NativeIoException: bind(..) failed: Address already in use
`

I'm assuming this is because my Minecraft server, that's running on the same machine, is already using the default port that BedrockConnect uses. Furthermore, I noticed in the Readme for the project that I cannot change the port for BedrockConnect if I want it to work for consoles. My question is: can I change the Minecraft server's port to a non-standard port, add it to my custom servers list, then launch BedrockConnect on the standard port?

**Comment by kmpoppe** [comment]
Hi @rstone13,

It's a known issue for over at [Mojang's Bug Tracker](https://bugs.mojang.com/browse/BDS-3989) that the Bedrock Server opens the default ports even though it's told otherwise.
It is a complete and utter mystery to me, how they cannot be bothered to fixing such a simple issue since it was reported in 1.14. It's a nuasence for every server admin because you can't have more than one world running on a server.

TL;DR: This is a bug, but not with Bedrock Connect ;)

**Comment by kmpoppe** [comment]
@Pugmatt, as I can't change the Wiki, I'd suggest you add this to the Troubleshooting Wiki Page and close this Issue with it.

K

```
### I can't start BedrockConnect on a Linux server that is also running the offical Minecraft Bedrock Server

For BedrockConnect to work correctly with any consoles, it needs to run on the default UDP/IP ports of the Bedrock Server, which are 19132 for IPv4 and 19133 for IPv6.

Since around August 2019 (Server version 1.14.32.1), the Minecraft Bedrock Server software opens these ports, even though the `server.properties` file configures different ports.

This conflict prevents either Server (Bedrock Server or BedrockConnect) that is started later to not work at all.

This is not an error of BedrockConnect but from the Minecraft Bedrock Server, that is bug-tracked at [Mojang's Jira](https://bugs.mojang.com/browse/BDS-3989). Watch that issue to get updates on what happens there.

While this bug is unresolved, there is NO way to run BedrockConnect on any server that has an actively running instance of the Minecraft Bedrock Server.
```

**Comment by Pugmatt** [comment]
> @Pugmatt, as I can't change the Wiki, I'd suggest you add this to the Troubleshooting Wiki Page and close this Issue with it.
> 
> K
> 
> ```
> ### I can't start BedrockConnect on a Linux server that is also running the offical Minecraft Bedrock Server
> 
> For BedrockConnect to work correctly with any consoles, it needs to run on the default UDP/IP ports of the Bedrock Server, which are 19132 for IPv4 and 19133 for IPv6.
> 
> Since around August 2019 (Server version 1.14.32.1), the Minecraft Bedrock Server software opens these ports, even though the `server.properties` file configures different ports.
> 
> This conflict prevents either Server (Bedrock Server or BedrockConnect) that is started later to not work at all.
> 
> This is not an error of BedrockConnect but from the Minecraft Bedrock Server, that is bug-tracked at [Mojang's Jira](https://bugs.mojang.com/browse/BDS-3989). Watch that issue to get updates on what happens there.
> 
> While this bug is unresolved, there is NO way to run BedrockConnect on any server that has an actively running instance of the Minecraft Bedrock Server.
> ```

Thanks for pointing this out. Updated the Troubleshooting page.
